src/CabanaPD_Boundary.hpp

```diff
@@ -112,6 +120,8 @@ struct BoundaryIndexSpace<MemorySpace, RegionBoundary>
                                   index_functor );
         }
     }
+
+    auto time() { return _timer.time(); };
```

Comment:
Why do we have the line below in CabanaPD_BodyTerm.hpp (Line 51) but not here?

auto timeInit() { return 0.0; };

Reply:
This is the setup for the boundary conditions, not the BC itself